src/db/mongodb/collection.rs

```diff
@@ -111,6 +112,13 @@ pub trait MongoDbCollectionExt: MongoDbCollection {
     ) -> Result<UpdateResult, Error> {
         self.with_type().replace_one(query, replacement, options).await
     }
+
+    /// Returns the number of documents in the collection.
+    #[cfg(feature = "rand")]
+    async fn len(&self) -> Result<usize, Error> {
```

Comment:
I would probably call the method `.count()` to avoid the clippy warning and to signal that this is potentially a more expensive computation.

Reply:
fine with me